spec/requests/ncr_work_order_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe 'proposals' do
+  include ReturnToHelper
+
+  describe "DISABLE_CLIENT_SLUGS" do
+    with_env_var("DISABLE_CLIENT_SLUGS", "ncr") do
+      it "disallows any request for disabled client_slug" do
+        user = create(:user, client_slug: "ncr")
```

Comment:
I wanted to test both in the abstract (`foo`) and in the concrete (`ncr`) because the story is specifically for NCR.